vars/CentOS_8.yml

```diff
@@ -5,3 +5,4 @@ blivet_package_list:
   - libblockdev-lvm
   - libblockdev-mdraid
   - libblockdev-swap
+  - vdo
```

Comment:
You should add `kmod-kvdo` here too, `vdo` package contains only the userspace tools.

Reply:
Fixed: `kmod-kvdo` added to `CentOS_8.yml`, `Fedora.yml` and `RHEL_8.yml`